tests/golang/invoke_test.go

```diff
@@ -69,29 +72,50 @@ func TestInvoke(t *testing.T) {
 	r.NoError(err)
 
 	t.Run("trace run should have appropriate data", func(t *testing.T) {
-		<-time.After(4 * time.Second)
+		// Wait for function run to complete
+		require.Eventually(t, func() bool {
+			return done
+		}, time.Second*4, time.Second)
```

Comment:
Could we use `WaitForRunStatus` instead?
https://github.com/inngest/inngest/blob/main/tests/golang/fn_output_test.go#L46